tests/bigint.rs

```diff
@@ -1092,3 +1092,29 @@ fn test_iter_product_generic() {
     assert_eq!(result, data.iter().product());
     assert_eq!(result, data.into_iter().product());
 }
+
+#[test]
+fn test_pow() {
+    let one = 1u32.to_bigint().unwrap();
+    let two = 2u32.to_bigint().unwrap();
+    let minus_two = -2i32.to_bigint().unwrap();
+    let four = 4u32.to_bigint().unwrap();
+    let eight = 8u32.to_bigint().unwrap();
```

Comment:
These can use `From` like `BigInt::from(8u32)` nowadays -- although I know many other tests are still doing it the hard way too. (We ought to clean that up...)

Reply:
:heavy_check_mark: